src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -621,4 +621,7 @@
     <value>Apply preferred 'using' placement preferences</value>
     <comment>'using' is a C# keyword and should not be localized</comment>
   </data>
+  <data name="Remove_in_keyword" xml:space="preserve">
+    <value>Remove 'in' keyword</value>
```

Comment:
check out the other resource strings.  There should be a way to mark `in` as `locked`.  this will tell the loc team to not localize that word when they do the different languages.